CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.6.0 (2017-06-9)
```

Comment:
Try using `09` format: `## 2.6.0 (2017-06-09)`. Also you will need to update this to the current date.